### PR TITLE
Fixes to Luke's first draft of code.

### PR DESCRIPTION
**Pull request by jjbenedict**
Fixes to Luke's first draft of code... AgI intr and AgI physics.

### CODE REVIEW

File: components/eam/src/physics/cam/agi_physics.F90 [hunk 1]
```diff
@@ -181,83 +189,96 @@ subroutine agi_scavenging(T, P, dissipation, sat_w, Nc, Nagi, esl, &
...
 			Ctb3 = factor/Map(6)
 
-            !Lama is defined in the second sentence of the first paragraph of Vuckovic et al 2022 page 4
+            ! Now compute constants C associated with thermophoresis and diffusiophoresis (Caro et al. equation C.9).
+            !   To compute Cthdf, first compute phiTH (Caro et al. equation 33) and A3 (Caro et al. equation 10), plus a few dependencies.
+            !Lama is defined in the first sentence of the first paragraph of Vuckovic et al 2022 page 4
             !just above eq 6
-            lama = lambdaAO*(Po/P(icol,k))*(T(icol,k)/To) !need to verify units of P, should be mb or convert to Pa)
+            P_hPa = P(icol,k) * 0.01_r8   ! compute air pressure in hPa (P is in units Pa)
+            lama = lambdaAO * (Po / P_hPa) * (T(icol,k) / Tref) !need to verify units of P, should be mb or convert to Pa)
```
Comment (vanroekel) [inline components/eam/src/physics/cam/agi_physics.F90]: @jjbenedict if Po is for sure in units of hPa can you delete the comment?

File: components/eam/src/physics/cam/agi_physics.F90 [hunk 2]
```diff
@@ -181,83 +189,96 @@ subroutine agi_scavenging(T, P, dissipation, sat_w, Nc, Nagi, esl, &
...
+			   mu_a = (1.718_r8 + 0.0049_r8*tdegC - 1.2E-5_r8 * tdegC**2.0_r8) * 1e-3_r8 !using 1e-3 due to conversion from poise to SI units
 			endif
 			nu_a = mu_a / rho ! kinematic viscosity is just dynamic visc / rho
 
             i = 1
             do m=-2,6
-               !Mass of aerosol particle (Map) is from Caro et al (2004) Eq C.3
-			   Map(i) = Nagi(icol,k)*Dagi**m*exp(m**2*log(sigAgi)**2/2.0_r8)
+               !Moments of aerosol particles (Map) are from Caro et al (2004) Eq C.3
+			   Map(i) = Nagi(icol,k) * Dagi**m * exp(m**2._r8 * log(sigAgi)**2._r8 / 2.0_r8)
```
Comment (vanroekel) [inline components/eam/src/physics/cam/agi_physics.F90:225]: I'm a bit confused by some places having spaces introduced between multiply and divide and not others.  Is there a reason for this?

File: components/eam/src/physics/cam/agi_physics.F90 (hunk 1 above)
Comment (jjbenedict) [reply components/eam/src/physics/cam/agi_physics.F90]: Short answer:  we can delete this comment.  I'm attempting another small PR for this...
Longer answer:  The only use of `Po` in agi_scavenging is where lama is computed.  On this line, the term (`Po / P_hPa`) is used.  This term -must- be unitless, and we arbitrarily decided to use both `Po` and `P_hPa` in units hPa.  So, `Po` can be defined in units hPa (and it already is).  `P_hPa` is computed as (`P*0.01`), and `P` is directly passed into agi_scavenging as `state%pmid`... so... if `state%pmid` is in units Pa (which I have to assume it is, right?), then we're good to go and the comment can be deleted.

File: components/eam/src/physics/cam/agi_physics.F90 (hunk 2 above)
Comment (jjbenedict) [reply components/eam/src/physics/cam/agi_physics.F90:225]: No meaningful reason other than more clarity.  In retrospect I should have just left it as is.

Comment (vanroekel) [reply components/eam/src/physics/cam/agi_physics.F90:225]: let's not worry about it, I'll just integrate

**Pull request merged**